### ISSUE TITLE
Clear Filter Icon on Excel Style Filtering has to be changed

### ISSUE THREAD

**Issue by DaniMarinov** [IgniteUI/igniteui-angular#5001]
## Description  
Clear filter Icon on Excel Style Filtering has to be changed. Currently, ESF filtering looks like that:
<img width="1440" alt="Screen Shot 2019-06-06 at 10 03 26 AM" src="https://user-images.githubusercontent.com/41998471/59013690-4ac32480-8843-11e9-82cc-1874c20f75c8.png">

As it is shown on 4.1.10 https://github.com/IgniteUI/igniteui-angular/wiki/Excel-Style-Filtering#41-end-user-experience , the label for "Clear filter" action has to be changed to:
**Clear Column Filters**

 * igniteui-angular version: 7.3.1
 * browser: 

## Steps to reproduce  

1. Go to the sample of ESF https://www.infragistics.com/products/ignite-ui-angular/angular/components/grid/excel_style_filtering.html 
2. Open ESF dialog
3. Check label at the Clear filter action

## Result  
<img width="1440" alt="Screen Shot 2019-06-06 at 10 03 26 AM" src="https://user-images.githubusercontent.com/41998471/59013949-fe2c1900-8843-11e9-9c61-c3823799b46c.png">

## Expected result  
<a href="https://ibb.co/RSvTR2F"><img src="https://i.ibb.co/RSvTR2F/Screen-Shot-2019-06-07-at-3-53-01-PM.png" alt="Screen-Shot-2019-06-07-at-3-53-01-PM" border="0"></a>

**Comment by gedinakova** [comment]
@DaniMarinov What is the reason behind this design? In MS Excel the option is called 'Clear filter for `ColumnName`'. 'Reset column filters' kind of implies we will reset to something, doesn't it? And the icon is a 'refresh' icon which may be translated that we will simply reapply the filter (if let's say the data records have changed for a certain reason).

**Comment by DaniMarinov** [comment]
The main problem comes from the X icon, which is mostly familiar from the inputs where X removes what's written so far in the input (the input value). But doesn't perform any other changes, such as reseting filters completely to the default state (both filter type and input value). In addition, X would be appropriate if it is used only for clearing the input value for specific filter. Reset implies not only clearing input values for all filters, but also removing all currently applied filters and turning them back to the default filter types.
In conclusion - Reset is more about affecting all applied filters, while Clear filters is rather about clearing only input value, but not affecting filter types.
